app/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Padland</string>
+    <string name="pref_header_general">Einstellungen</string>
+    <string name="title_activity_settings">Einstellungen</string>
+    <string name="action_settings">Einstellungen</string>
+    <string name="go_pad">Erstellen oder anzeigen</string>
+    <string name="title_activity_pad_view">Padland-Anzeige</string>
+    <string name="write_pad_name">Pad-Name</string>
+    <string name="action_newpad">Dokument hinzufügen</string>
+    <string name="action_go_to_list">Zurück zur Pad-Liste</string>
+    <string name="title_activity_new_pad">Pad erstellen</string>
+    <string name="new_pad_explanation">Gib den Pad-Name ein. Wenn es nicht existiert, wird es erstellt; andernfalls wird es dich zum Pad bringen.</string>
```

Comment:
BTW: Should the  [original string](https://github.com/rugk/padland/blob/patch-1/app/src/main/res/values/strings.xml) really mean "Introduce the pad name"? I think "Enter the pad name" is meant here, is not it?

This is the way I translated it.